applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -361,14 +303,16 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 static void syncable_cb(struct bt_bap_broadcast_sink *sink, bool encrypted)
 {
 	int ret;
-	static uint8_t bis_encryption_key[16];
+	static uint8_t bis_encryption_key[16] = {0};
```

Comment:
Consider using `BT_ISO_BROADCAST_CODE_SIZE` or `BT_AUDIO_BROADCAST_CODE_SIZE` instead of `[16]`

Reply:
Done